src/mlpack/methods/random_forest/random_forest_impl.hpp

```diff
@@ -251,29 +273,32 @@ double RandomForest<
          const double minimumGainSplit,
          const size_t maximumDepth,
          const bool warmStart,
-         DimensionSelectionType dimensionSelector)
+         DimensionSelectionType dimensionSelector,
+         BootstrapType bootstrap)
 {
   // Pass off to Train().
   data::DatasetInfo datasetInfo; // Ignored by Train().
-  return Train<false, false>(dataset, datasetInfo, labels, numClasses, weights,
+  return Train<true, false>(dataset, datasetInfo, labels, numClasses, weights,
```

Comment:
@rcurtin , was this an oversight in the original implementation? I see that the outer `Train` function has the `weights` parameter, so I'd expect that the inner `Train` function uses the first template argument as `true`. What do you think?

Reply:
It was an oversight!  Thank you for catching it. :+1: